test/new-e2e/tests/windows/remotesuite.go

```diff
@@ -0,0 +1,142 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package windows
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/stretchr/testify/require"
+)
+
+// RemoteSuite is a helper struct to run tests on a remote host
+type RemoteSuite struct {
```

Comment:
I renamed the class to something hopefully less ambiguous.  I would not like to move it, because the very next test in the queue will also consume this, which is why I did it this way.  If I move it into the suite, I'll just have to move it right back out.